chapters/VK_KHR_performance_query/features.txt

```diff
@@ -11,52 +11,22 @@ as:
 include::{generated}/api/structs/VkPhysicalDevicePerformanceQueryFeaturesKHR.txt[]
 
 The members of the sname:VkPhysicalDevicePerformanceQueryFeaturesKHR
-structure describe the following implementation-dependent features:
-
-  * pname:sType is the type of this structure.
-  * pname:pNext is `NULL` or a pointer to a structure extending this
-    structure.
-  * pname:performanceCounterQueryPools is ename:VK_TRUE if the physical
-    device supports performance counter query pools.
-  * pname:performanceCounterMultipleQueryPools is ename:VK_TRUE` if the
-    physical device supports using multiple performance query pools in a
+structure describe the following features:
+
+  * [[features-performanceCounterQueryPools]]
```

Comment:
We want to describe all the struct members here, including sType/pNext.

Reply:
done
I used `VkPhysicalDeviceShaderAtomicInt64Features` as a template instead.